### PR TITLE
Add gas, volume and energy in 4 bytes

### PR DESCRIPTION
**Pull request by Ernst79**
Add gas, volume and energy in 4 bytes

### CODE REVIEW

File: src/format.html [hunk 1]
```diff
@@ -682,6 +712,21 @@ <h4 id="sensor-data">Sensor data</h4>
           <code>V</code>
         </td>
       </tr>
+      <tr>
+        <td>
+          <code>0x4E</code>
+        </td>
+        <td>volume</td>
```
Comment (balloob) [inline src/format.html:719]: Does it make sense to have a generic volume? Should it be more precise and specify a volume of "what"

Comment (Ernst79) [reply src/format.html:719]: By making it general, people can use it for whatever they want, keeping it flexible to use. But I can add water as extra object type, as that will be a very commonly used.

Comment (balloob) [reply src/format.html:719]: We have `gas` as a type. So I would expect `water` to be one too.

Comment (Ernst79) [reply src/format.html:719]: Ok, I'll add waters in a separate PR. I've created an issue such that I don't forget.

Comment (balloob) [reply src/format.html:719]: do you still want to include volume? Or would just the water sensor cover the current use cases

Comment (Ernst79) [reply src/format.html:719]: Yes, let’s keep it. Can be useful for other liquids, beer, oil,… We already had a volume sensor with less bytes as well.

**Pull request merged**